### PR TITLE
Use tslint instead of prettier as the default check before tests run

### PR DESCRIPTION
**Pull request by snario**
Since we use [this plugin](https://github.com/prettier/tslint-plugin-prettier#tslint-plugin-prettier) `tslint` actually uses `prettier` under the hood, so these checks are more extensive. This also fixes an issue where the `wallet` was inheriting properties from a root `.prettierrc` which `web3torrent` and `embedded-wallet` are inheriting their linting rules from.

TODO: Standardizing our TypeScript code styles across all packages.

### CODE REVIEW

File: packages/nitro-protocol/package.json [hunk 1]
```diff
@@ -35,7 +35,7 @@
     "build:ci": "yarn build",
     "test:ci": "yarn test",
     "test:typescript": "npx jest --testPathIgnorePatterns='test/contracts' --reporters='default'",
-    "test": "yarn prettier:check && yarn clearContracts && npx ganache-then-jest --runInBand",
+    "test": "yarn lint:check && yarn clearContracts && npx ganache-then-jest --runInBand",
```
Comment (cf19drofxots) [inline packages/nitro-protocol/package.json]: Can we decouple linting from testing? They are separate concerns.

File: packages/wallet/package.json [hunk 2]
```diff
@@ -15,12 +15,14 @@
     "build:typescript": "npx tsc",
     "build:webpack": "CI=false node scripts/build.js",
     "prepare": "npx tsc",
-    "test": "run-s prettier:check 'test:app --all'",
-    "test:ci": "GANACHE_PORT=8503 yarn run prettier:check && yarn run test:ci:contracts && yarn run test:ci:app && yarn build",
+    "test": "run-s lint:check 'test:app --all'",
```
Comment (cf19drofxots) [inline packages/wallet/package.json]: Ditto earlier decoupling comment

File: packages/nitro-protocol/package.json (hunk 1 above)
Comment (snario) [reply packages/nitro-protocol/package.json]: Going to do this in a separate PR

File: packages/wallet/package.json (hunk 2 above)
Comment (snario) [reply packages/wallet/package.json]: Going to do this in a separate PR

**Pull request merged**